besu/src/main/java/org/hyperledger/besu/cli/options/EthProtocolOptions.java

```diff
@@ -90,13 +88,6 @@ public class EthProtocolOptions implements CLIOptions<EthProtocolConfiguration>
   private PositiveNumber maxGetPooledTransactions =
       PositiveNumber.fromInt(EthProtocolConfiguration.DEFAULT_MAX_GET_POOLED_TRANSACTIONS);
 
-  @CommandLine.Option(
-      names = {LEGACY_ETH_64_FORK_ID_ENABLED},
-      paramLabel = "<Boolean>",
-      description = "Enable the legacy Eth/64 fork id. (default: ${DEFAULT-VALUE})")
-  private Boolean legacyEth64ForkIdEnabled =
```

Comment:
maybe it's worth keeping just the CLI option as a no-op for one release so we don't break existing configs? Make it a hidden option, flag it as a upcoming breaking change. The rest of the PR is non-breaking?

Reply:
done